File: 2274-keep-multiplying-found-values-by-two/keep-multiplying-found-values-by-two.java
class Solution {
    public int findFinalValue(int[] nums, int original) {
        int found = 0;
        while(found != 1){
            for(int i = 0; i<nums.length;i++){
                if (nums[i] == original){
                    found = 0;
                    original *= 2;
                    break;
                }
                found = 1;
            }
        }
        return original;
    }
}

File: 2274-keep-multiplying-found-values-by-two/keep-multiplying-found-values-by-two.py
class Solution(object):
    def findFinalValue(self, nums, original):
        found = 0
        while found != 1:
            for each in nums:
                if each == original:
                    found = 0
                    original *=2
                    break
                found = 1
        return original

            
        